src/frontend/Sider/Personoversikt/Behandlingsoversikt/OpprettNyBehandling/OpprettOrdinærBehandling.tsx

```diff
@@ -130,6 +177,36 @@ const OpprettOrdinærBehandling: React.FC<Props> = ({
                     toDate={new Date()}
                 />
             )}
+            {årsak === BehandlingÅrsak.NYE_OPPLYSNINGER && (
+                <>
+                    <Select label={'Kilde til opplysninger'} onChange={endreKilde}>
+                        <option value={''}>-Velg kilde-</option>
+                        <option value={NyeOpplysningerKilde.MODIA}>Modia</option>
+                        <option value={NyeOpplysningerKilde.GOSYS}>Gosys</option>
+                        <option value={NyeOpplysningerKilde.ETTERSENDING}>Ettersending</option>
+                        <option value={NyeOpplysningerKilde.OPPFØLGINGSLISTE}>
+                            Oppfølgingsliste
+                        </option>
+                        <option value={NyeOpplysningerKilde.ANNET}>Annet</option>
+                    </Select>
```

Comment:
Kunde ev vurdert en key-map her for at nye typer automatisk blir lagt til i listen dersom man legger til et enum-verdi, på lik måte som for `NyeOpplysningerEndring`
```typescript
{Object.keys(NyeOpplysningerKilde).map((kilde) => (
                            <option key={kilde} value={kilde}>
                                {kildeTilTekst[kilde]}
                            </option>
                        ))}
```